website/docs/d/vpc_ipam_pools.html.markdown

```diff
@@ -0,0 +1,79 @@
+---
+subcategory: "VPC IPAM (IP Address Manager)"
+layout: "aws"
+page_title: "AWS: aws_vpc_ipam_pools"
+description: |-
+    Returns details about the first IPAM pool that matches search parameters provided.
+---
+
+# Data Source: aws_vpc_ipam_pools
+
+`aws_vpc_ipam_pools` provides details IPAM pools.
+
+This resource can prove useful when an ipam pool was created in another root
+module and you need the pool's id as an input variable. For example, pools
+can be shared via RAM and used to create vpcs with CIDRs from that pool.
+
+## Example Usage
+
+The following example shows an account that has only 1 pool, perhaps shared
+via RAM, and using that pool id to create a VPC with a CIDR derived from
+AWS IPAM.
```

Comment:
The example below should show how to get the ID from the data source to match this description

Reply:
i can safely remove this